pwdtk/auth_backends.py

```diff
@@ -14,19 +14,39 @@
 logger = logging.getLogger(__name__)
 
 
-class PwdtkLockedException(Exception):
-    """ custom exception """
+class PwdtkBaseException(Exception):
 
     def __init__(self, pwdtk_data):
 
         self.pwdtk_data = pwdtk_data
 
 
+class PwdtkLockedException(PwdtkBaseException):
+    """ custom exception """
+
+
+class PwdtkForceRenewException(PwdtkBaseException):
+    """ custom exception """
+
+
 class PwdtkBackend(ModelBackend):
     """
     Authenticates against settings.AUTH_USER_MODEL.
     """
+    def compute_failed_login(self, pwdtk_data):
+        if PwdtkSettings.PWDTK_USER_FAILURE_LIMIT is None:
+            return None
+        if (pwdtk_data.failed_logins+1 >=
+           PwdtkSettings.PWDTK_USER_FAILURE_LIMIT):
+            pwdtk_data.set_locked(pwdtk_data.failed_logins+1)
```

Comment:
why are line 39 and 40 not on the same line? 
really longer than 80 caracters???